tests/dispatch_io_net.c

```diff
@@ -94,11 +92,13 @@ main(int argc, char** argv)
 
 		// Read from the socket and compare the contents are what we expect
 
+		const char *path = argv[2];
 		fd = open(path, O_RDONLY);
 		if (fd == -1) {
 			test_errno("client-open", errno, 0);
 			test_stop();
 		}
+		dispatch_test_release_large_file(path);
```

Comment:
It looks odd that you release a file here without having created it. I had to read the rest of the source file to find out why this is correct. Some comments here about how this file is created would be useful.